### PR TITLE
add websocketpp/0.8.2

### PR DESCRIPTION
**Pull request by prince-chrismc**
Specify library name and version:  **lib/1.0**

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/wiki#how-to-submit-a-pull-request) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by prince-chrismc** [comment]
RIP wasnt paying attention... crap force pushed the wrong branch.

File: recipes/websocketpp/all/conanfile.py [hunk 1]
```diff
@@ -34,18 +34,20 @@ def source(self):
...
+        # We have to copy the headers manually, since the upstream cmake.install() step doesn't do so.
         self.copy(pattern="*.hpp", dst="include/websocketpp", src=self._source_subfolder + "/websocketpp")
 
     def package_info(self):
         if self.options.asio == "standalone":
             self.cpp_info.defines.extend(["ASIO_STANDALONE", "_WEBSOCKETPP_CPP11_STL_"])
 
-    def package_id(self):
+    def package_id(self):	
         self.info.header_only()
```
Comment (prince-chrismc) [inline recipes/websocketpp/all/conanfile.py:52]: Do these too declarations conflict? should the options/defines the package id?

Comment (uilianries) [reply recipes/websocketpp/all/conanfile.py:52]: there is a trailing space, please, remove it.

File: recipes/websocketpp/all/conanfile.py [hunk 2]
```diff
@@ -34,18 +34,20 @@ def source(self):
...
-        # We have to copy the headers manually, since the current cmake.install() step in the 0.8.1 release doesn't do so.
+        # We have to copy the headers manually, since the upstream cmake.install() step doesn't do so.
         self.copy(pattern="*.hpp", dst="include/websocketpp", src=self._source_subfolder + "/websocketpp")
 
     def package_info(self):
         if self.options.asio == "standalone":
             self.cpp_info.defines.extend(["ASIO_STANDALONE", "_WEBSOCKETPP_CPP11_STL_"])
 
-    def package_id(self):
+    def package_id(self):
```
Comment (prince-chrismc) [inline recipes/websocketpp/all/conanfile.py:51]:
```suggestion
    def package_id(self):
```

**Pull request merged**